packages/cli/src/commands/login.ts

```diff
@@ -66,23 +35,24 @@ export default class Login extends BaseCommand {
     this.log('Welcome to the Checkly CLI')
   }
 
-  async run (): Promise<void> {
-    const { flags } = await this.parse(Login)
-    const { 'api-key': apiKey, 'account-id': accountId } = flags
-
-    await this._checkExistingCredentials()
+  private selectAccount = async (accounts: Array<Account>): Promise<Account|undefined> => {
+    if (accounts.length === 1) {
+      return accounts[0]
+    }
 
-    if (apiKey) {
-      if (!accountId) {
-        throw new Error('The flag --account-id (-i) is required when using --api-key (-k)')
-      }
+    const { accountName } = await prompts({
+      name: 'accountName',
+      type: 'select',
+      choices: accounts.map(({ name }) => ({ title: name, value: name })),
```

Comment:
We can simplify this further in a new PR and just pass the whole account as value instead of just the name